### DISCUSSION TITLE
Other chartjs components - trendline

### DISCUSSION THREAD

**Discussion by ledpup** [Megabit/Blazorise#3231] [Q&A] [answered]
Hi all

I'm super new to Blazor and Blazorise but making great progress with the chartjs component and other elements. Really good.

I'd like to implement the trendline from https://www.npmjs.com/package/chartjs-plugin-trendline but it doesn't look like a Blazorise version has been done.

Does anyone have any suggestions on how to approach implementing it? Can just hooking into the JS work? Write a Blazor wrapper for it?

Thanks
Patrick

**Comment by stsrki** [comment, marked as answer]
We just recently implemented Chart.js 3.X so it should be possible to include support for the trendline plugin. If you wish to contribute everything is already merged in the `master` branch. And you can look at our chart streaming plugin for reference.
